### ISSUE TITLE
Reveal SecretKey for power users

### ISSUE THREAD

**Issue by friedger** [blockstack/stacks-wallet#920]
There could be a switch for power users to advanced mode, that can reveal the current Secret Key in the settings.

**Comment by copyrightbreaker** [comment]
Yes, I currently am locked out of accessing my tokens since I lost my seed phrase. Would be a much benefited edition if this were added.

**Comment by kzk4** [comment]
I need to see my SEED PHRASE on Hiro Wallet ! please fix it

**Comment by markmhx** [comment]
This has been requested here as well on Reddit: https://www.reddit.com/r/stacks/comments/w32nrz/i_want_to_like_stacks_but/?utm_medium=android_app&utm_source=share

**Comment by Cactii1** [comment]
I'd like to see this. Personally I don't like browser based wallets. Maybe I'm just paranoid but this type of setup seems way less secure than a desktop wallet.

**Comment by Cactii1** [comment]
Even on the Hiro web site it says that being able to see the secret key is a supported feature:

https://www.hiro.so/wallet/whats-the-difference-between-the-browser-extension-and-the-desktop-app

At a minimum this should be updated to not say something that's not true.

**Comment by GAD2511** [comment]
Basic view: Not everyone interacts with apps, they use desktop wallets just to store their asset. Also, not everyone trusts web wallets.
Result: No need to install a web wallet.

Reveal the Secret key with password entry will be so helpful for this kind of user and practically will benefit all users.

**Comment by markmhx** [comment]
> Even on the Hiro web site it says that being able to see the secret key is a supported feature:

Thanks for the heads up. I've corrected that comparison table to reflect that it's _not_ yet supported by desktop.
